jax/dtypes.py

```diff
@@ -218,6 +218,9 @@ def is_python_scalar(x):
   except AttributeError:
     return type(x) in python_scalar_dtypes
 
+def is_scalar(x):
+  return is_python_scalar(x) or isinstance(x, onp.generic)
```

Comment:
This is an (unrelated) breaking change in terms of how JAX handles NumPy scalars